### DISCUSSION TITLE
请问可以把 Caddy 作为前段，转发 Hysteria2 链接至 sing-box 吗？

### DISCUSSION THREAD

**Discussion by gsgqgyq** [lxhao61/integrated-examples#204] [Q&A] [answered]
rt，目前在用 sing-box 进行配置，想同时启用 Hysteria2，请问可以共用 443 端口实现转发吗？

**Comment by lxhao61** [comment, marked as answer]
不能。
Caddy 仅支持反代 WebSocket（含 HttpUpgrade）、H2C（含 gRPC） 应用；Caddy  SNI 目前仅支持基于域名的 TCP 端口分流。
Caddy 可使用 TCP 443 端口， Hysteria2 可使用 UDP 443 端口来实现所谓共用 443 端口。
